2019-05-18 17:26:05.387653: I tensorflow/core/platform/cpu_feature_guard.cc:141] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2 FMA
2019-05-18 17:26:05.780167: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1405] Found device 0 with properties: 
name: TITAN V major: 7 minor: 0 memoryClockRate(GHz): 1.455
pciBusID: 0000:02:00.0
totalMemory: 11.75GiB freeMemory: 10.03GiB
2019-05-18 17:26:05.780248: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1484] Adding visible gpu devices: 0
2019-05-18 17:26:06.660258: I tensorflow/core/common_runtime/gpu/gpu_device.cc:965] Device interconnect StreamExecutor with strength 1 edge matrix:
2019-05-18 17:26:06.660317: I tensorflow/core/common_runtime/gpu/gpu_device.cc:971]      0 
2019-05-18 17:26:06.660325: I tensorflow/core/common_runtime/gpu/gpu_device.cc:984] 0:   N 
2019-05-18 17:26:06.660716: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1097] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 9672 MB memory) -> physical GPU (device: 0, name: TITAN V, pci bus id: 0000:02:00.0, compute capability: 7.0)
Generator will have 200378880 samples in 97841 batches
Features have shape 64; targets have shape 65
Generator will have 200378880 samples in 97841 batches
Features have shape 64; targets have shape 65
Epoch 1/25
 - 1699s - loss: 10449988.6213 - mean_squared_error: 10449988.6213 - val_loss: 1592.3522 - val_mean_squared_error: 1592.3522
Epoch 2/25
 - 1654s - loss: 28612.2188 - mean_squared_error: 28612.2188 - val_loss: 344.7998 - val_mean_squared_error: 344.7998
Epoch 3/25
 - 1420s - loss: 1223.9170 - mean_squared_error: 1223.9170 - val_loss: 2533.0857 - val_mean_squared_error: 2533.0857
Epoch 4/25
 - 1680s - loss: 258972.8946 - mean_squared_error: 258972.8946 - val_loss: 190.7980 - val_mean_squared_error: 190.7980
Epoch 5/25
 - 1481s - loss: 1832.8912 - mean_squared_error: 1832.8912 - val_loss: 86.2769 - val_mean_squared_error: 86.2769
Epoch 6/25
 - 1639s - loss: 2673.1965 - mean_squared_error: 2673.1965 - val_loss: 0.0123 - val_mean_squared_error: 0.0123
Epoch 7/25
 - 1568s - loss: 1253.5780 - mean_squared_error: 1253.5780 - val_loss: 0.1820 - val_mean_squared_error: 0.1820
Epoch 8/25
 - 1587s - loss: 13288.6667 - mean_squared_error: 13288.6667 - val_loss: 0.1211 - val_mean_squared_error: 0.1211
Epoch 9/25
 - 1463s - loss: 202.8927 - mean_squared_error: 202.8927 - val_loss: 77.1342 - val_mean_squared_error: 77.1342
Epoch 10/25
 - 1346s - loss: 1244.7171 - mean_squared_error: 1244.7171 - val_loss: 0.2562 - val_mean_squared_error: 0.2562
Epoch 11/25
 - 1125s - loss: 7389.7208 - mean_squared_error: 7389.7208 - val_loss: 6.5499 - val_mean_squared_error: 6.5499
Epoch 12/25
 - 1233s - loss: 11825.5453 - mean_squared_error: 11825.5453 - val_loss: 27.0426 - val_mean_squared_error: 27.0426
Epoch 13/25
 - 1301s - loss: 13610.3078 - mean_squared_error: 13610.3078 - val_loss: 10.3201 - val_mean_squared_error: 10.3201
Epoch 14/25
 - 1262s - loss: 27179.6802 - mean_squared_error: 27179.6802 - val_loss: 0.4223 - val_mean_squared_error: 0.4223
Epoch 15/25
 - 1233s - loss: 230.3680 - mean_squared_error: 230.3680 - val_loss: 0.5651 - val_mean_squared_error: 0.5651
Epoch 16/25
 - 1219s - loss: 2037.6270 - mean_squared_error: 2037.6270 - val_loss: 0.6529 - val_mean_squared_error: 0.6529
/pkg/python/3.6.1-centos7/lib/python3.6/site-packages/h5py/__init__.py:34: FutureWarning: Conversion of the second argument of issubdtype from `float` to `np.floating` is deprecated. In future, it will be treated as `np.float64 == np.dtype(float).type`.
  from ._conv import register_converters as _register_converters
Using TensorFlow backend.
Done with trial 22
